src/MoviesContainer/MoviesContainer.test.js

```diff
@@ -1,14 +1,51 @@
 import { render, screen, fireEvent } from '@testing-library/react';
 import '@testing-library/jest-dom';
-import MovieContainer from './MovieContainer';
+import MoviesContainer from './MoviesContainer';
 
 describe('MovieContainer', () => {
-  it('should return movieCard on page load', () => {
-    // setup
-    // render the movieCard, with mocked const(s) of movies and mock functions of movieClicked displaySingleMovie
-    // execution
-    // 
-    // assertion
-    // expect the const to be in the document
+  it('should return movie cards on page load', () => {
```

Comment:
This is such a long one! But I'm excited to hear the response from Leta and Will about if we need to test displayMovieDetails on this one!